webapp/public/index.html

```diff
@@ -21,22 +21,28 @@
       name="description"
       content="The dashboard is an open source project for monitoring network data and block producer infrastructure for EOSIO blockchains."
     />
-    <meta property="image" content="%PUBLIC_URL%/maskable_icon.png" />
+    <meta
+      property="image"
+      content="https://mainnet.eosio.online/maskable_icon.png"
+    />
     <meta property="og:type" content="website" />
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content="%PUBLIC_URL%/" />
     <meta
       property="og:title"
       content="Network Monitor and Infrastructure Dashboard for EOSIO blockchain networks"
     />
-    <meta property="og:image" content="%PUBLIC_URL%/maskable_icon.png" />
+    <meta
+      property="og:image"
+      content="https://mainnet.eosio.online/maskable_icon.png"
+    />
     <meta content="image/*" property="og:image:type" />
     <meta property="og:url" content="%PUBLIC_URL%/" />
     <meta property="og:site_name" content="Dashboard" />
     <meta
       property="og:description"
       content="The dashboard is an open source project for monitoring network data and block producer infrastructure for EOSIO blockchains."
```

Comment:
EOSIO dashboard is a network monitor featuring real-time data on block producer nodes and blockchain infrastructure for multiple EOSIO chains.